src/cuda/version_script.lds

```diff
@@ -0,0 +1,9 @@
+#_init and _fini should be local
+VERS_1.0 {
+  global:
+    GetInterface;    
```

Comment:
Would we ever need to expose any other functions in the future? Are we able to deal with the visibility from inside the code using ```_attribute__((visibility("default")))```

Reply:
We need to update for all platforms each time. For the providers it should only ever be one function, since it returns an interface and we can change that interface arbitrarily. So this should be fine.